synapse/storage/databases/main/event_push_actions.py

```diff
@@ -274,15 +274,6 @@ def __init__(
                 self._clear_old_push_actions_staging, 30 * 60 * 1000
             )
 
-        self.db_pool.updates.register_background_index_update(
-            "event_push_summary_unique_index",
-            index_name="event_push_summary_unique_index",
-            table="event_push_summary",
-            columns=["user_id", "room_id"],
-            unique=True,
-            replaces_index="event_push_summary_user_rm",
-        )
-
```

Comment:
Sanity check: what does the background update logic do if it tries to run an update but there's no handler for it?

Oh, the migration deletes the update in question, so it doesn't matter. :+1:

Reply:
An error gets raised if it cannot find the background update, but as you've said we're deleting it so it doesn't matter.